### ISSUE TITLE
Error: Please add a @NgModule annotation in 1.3.0

### ISSUE THREAD

**Issue by ClemensSchneider** [VadimDez/ng2-img-fallback#24]
Seems like the latest `1.3.0` release is missing some metadata as I get the following error:

```
ERROR in : Unexpected value 'ImgFallbackModule in <path-to-project>/node_modules/ngx-img-fallback/src/app/modules/ngx-img-fallback/ngx-img-fallback.module.d.ts' imported by the module 'SharedModule in <path-to-project>/src/app/shared/shared.module.ts'. Please add a @NgModule annotation.

```

Downgrading to `1.2.0` fixes this issue.

**Comment by VadimDez** [comment]
Are you using ng-cli?
Or wha's your setup?
Which angular version?

**Comment by ClemensSchneider** [comment]
```
Angular CLI: 1.6.7
Node: 8.7.0
OS: win32 x64
Angular: 5.2.3
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

@angular/cdk: 5.1.1
@angular/cli: 1.6.7
@angular/material: 5.1.1
@angular-devkit/build-optimizer: 0.0.42
@angular-devkit/core: 0.0.29
@angular-devkit/schematics: 0.0.52
@ngtools/json-schema: 1.1.0
@ngtools/webpack: 1.9.7
@schematics/angular: 0.1.17
typescript: 2.6.2
webpack: 3.10.0
```

**Comment by ClemensSchneider** [comment]
But same with @angular 5.0.0, cli 1.5.5:

```
Angular CLI: 1.5.5
Node: 8.7.0
OS: win32 x64
Angular: 5.0.0
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

@angular/cdk: 5.0.1
@angular/cli: 1.5.5
@angular/material: 5.0.1
@angular-devkit/build-optimizer: 0.0.36
@angular-devkit/core: 0.0.22
@angular-devkit/schematics: 0.0.42
@ngtools/json-schema: 1.1.0
@ngtools/webpack: 1.8.5
@schematics/angular: 0.1.11
@schematics/schematics: 0.0.11
typescript: 2.4.2
webpack: 3.8.1
```

**Comment by VadimDez** [comment]
The issue is fixed in `1.3.1` release
